ui/src/components/Entity/EntityInfoTags.scss

```diff
@@ -0,0 +1,7 @@
+@import "src/app/variables.scss";
+
+.EntityInfo {
+  .pt-text-muted {
+    margin-top: $aleph-grid-size * 0.5 !important;
```

Comment:
That seems super dangerous, we use .pt-text-muted for other things. What is it meant to apply to?